earth_enterprise/src/server/wsgi/serve/push/search/util/search_schema_parser.py

```diff
@@ -123,9 +123,9 @@ def Parse(self, search_file, table_name, file_prefix=None):
       logger.info("File prefix is '%s'", file_prefix)
     self.__StartDocument()
     try:
-      context = xml.etree.cElementTree.iterparse(search_file,
+      context = ET.iterparse(search_file,
                                                  SearchSchemaParser.EVENTS)
-    except xml.etree.cElementTree.ParseError, e:
+    except ET.ParseError, e:
```

Comment:
This may be an issue. I don't think `cElementTree` includes `ParseError`, though I haven't had the chance to check that yet. For some reason, `ElementTree` does include it, but I don't know if there are problems switching between the two versions of `*ElementTree`.